packages/react/src/Bento/BentoItem.features.stories.tsx

```diff
@@ -0,0 +1,455 @@
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual>
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+HeadingPaddingSpacious.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const VisualPaddingCondensed: StoryFn<typeof Bento> = () => (
+  <Bento.Item>
+    <Bento.Content>
+      <Heading as="h3">Heading</Heading>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual padding="condensed">
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+VisualPaddingCondensed.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const VisualPaddingNormal: StoryFn<typeof Bento> = () => (
+  <Bento.Item>
+    <Bento.Content>
+      <Heading as="h3">Heading</Heading>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual padding="normal">
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+VisualPaddingNormal.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const VisualPaddingSpacious: StoryFn<typeof Bento> = () => (
+  <Bento.Item>
+    <Bento.Content>
+      <Heading as="h3">Heading</Heading>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual padding="spacious">
+      <img className="test" alt="placeholder, blank area with an gray background color" src={placeholderImage} />
+    </Bento.Visual>
+  </Bento.Item>
+)
+
+VisualPaddingSpacious.parameters = {
+  viewport: {
+    defaultViewport: 'iphonexr',
+  },
+}
+
+export const VisualAsBackground: StoryFn<typeof Bento> = () => (
+  <Bento.Item visualAsBackground colorMode="dark">
+    <Bento.Content>
+      <Heading as="h3">Heading</Heading>
+      <Text>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit ullamcorper id. Aliquam luctus sed turpis
+        felis nam pulvinar risus elementum.
+      </Text>
+      <Link href="#">Call to action</Link>
+    </Bento.Content>
+    <Bento.Visual>
+      <img className="test" alt="A random avatar" src={universeImage} />
```

Comment:
Appreciate this is just an example, but can we source a better image that doesn't interfere with text legibility here?

![Screenshot 2023-08-24 at 09 53 36](https://github.com/primer/brand/assets/13340707/fa9eceaf-ae5d-451c-b1e6-77bfc1b9761e)

Our Storybook is usually referenced for A11Y reviews, and this is something that'll get immediately flagged but is easily avoidable.

Reply:
This has been updated in this [SHA](https://github.com/primer/brand/pull/390/commits/015e64c3e342f101c896f8259daf2c49feb49558) to use the placeholder image which will have a better contrast ratio.